config/config.go

```diff
@@ -64,6 +64,9 @@ type Server struct {
 	// DebugPort represents debug server port.
 	DebugPort int `yaml:"debug_port"`
 
+	// DebugPolicyCachePath represents the policy cache debug path
+	DebugPolicyCachePath string `yaml:"debug_policy_cache_path"`
```

Comment:
maybe this is too much for user?
I think it is easier for them to follow the hard coded endpoint.